src/main/webapp/pages/UserPage.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="../css/style.css">
+    <link rel="stylesheet" href="../css/style-userpage.css">
+    <script src="../javascript/RecipeModal.js"></script>
+  </head>
+  <body>
+    <div class="content">
+      <!-- Planner content -->
+      <div>
+        <h2>Planner</h2>
+        <div class="recipes">
+        </div>
+      </div>
+      <!-- Cookbook content -->
+      <div>
+        <h2>Cookbook</h3>
+        <div class="recipes">
+        </div>
+      </div>
+      <!-- User Recipes content -->
+      <div>
+        <h2>My Recipes</h2>
+        <button id="openModal" type="button" onClick="openModal()">Add Recipe</button>
+        <div id="recipe-modal">
+        <span onclick="closeModal()" class="close">&times;</span>
+          <form id="recipe-form" method="POST" action="/new-recipe">
+            <h1 class="center">Create Recipe</h1>
+            <label for="recipeName" required>Recipe Name:</label><br>
+            <input type="text" id="recipeName" name="recipeName">
+            <br><br>
+            <label for="tags">Tags:<br><i>Each tag should be separated by a comma.</i></label><br>
+            <input type="text" id="tags" name="tags">
+            <br><br>
+            <label for="ingredients" required>Ingredients:<br><i>Each ingredient should be on its own line.</i></label><br>
+            <textarea id="ingredients" name="ingredients" rows="4"></textarea>
+            <br><br>
+            <label for="steps" required>Steps:<br><i>Each step should be on its own line. No need to number them.</i></label><br>
+            <textarea id="steps" name="steps" rows="4"></textarea>
+            <br><br>
+            <label for="privacy">Choose a privacy setting:</label>
+            <select id="privacy" name="privacy">
+            <option value="private">private</option>
+            <!-- <option value="public">public</option> -->
```

Comment:
Should this be un-commented? Or is it not implemented yet?

Reply:
it's not implemented yet